main.cpp

```diff
@@ -122,394 +172,476 @@ struct CarWash
-    //amplify signal
     void doAmp(int channelNum = 0);
-    //produce noise
     bool noiseReduction(int filterNum = 3);
 };
-/*
-Thing 9) body
-5 properties:
-    1) height
-    2) width
-    3) depth
-    4) colour
-    5) material
-3 things it can do:
-    1) hold internal parts
-    2) protect circuit
-    3) collect dust
- */
+
 struct Body
 {
-    //height
     float height = 6.5f;
-    //width
     float width = 15.0f;
-    //depth
     float depth = 10.5f;
-    //colour
     char colour = 'B';
-    //material
     char material = 'W';
     
-     //hold internal parts
-     bool checkTheBolt(int circuitId = 1);
-     //protect circuit
-     bool checkShortCircuit(int circuitId = 1);
-     //collect dust
-     void alarmOverDust(float time = 10);
+    bool checkTheBolt(int circuitId = 1);
+    bool checkShortCircuit(int circuitId = 1);
+    void alarmOverDust(float time = 10);
 };
-/*
-Thing 10) Audio monitor controller
-5 properties:
-    1) DAC
-    2) power unit
-    3) VCA
-    4) headphone amplifier
-    5) body
-3 things it can do:
-    1) control volume
-    2) set input sources
-    3) toggle crosfeed effect
- */
 
 struct MonitorController
 {
-    //left channel DAC
     DAC DACLeft;
-    //right channel DAC
     DAC DACRight;
-    //power supply unit
     PowerUnit PSU0;
-    //volt controlled attenuator
     VCA VCA0;
-    //left channle amplifier
     HeadphoneAmp AmpLeft;
-    //right channle amplifier
     HeadphoneAmp AmpRight;
-    //body of controller
     Body body0;
 
-    //control volume
     void setVol(int amount = 0);
-    //set input sources
     bool selectSource(int sourceNum, bool status=0);
-    //toggle crosfeed effect
     bool toggleCrossfeed(bool status = 0);
 };
-   
+
+void CanPlant::OfficeWorker::makeCall(int customerId, float time)
+{
+    SipProvider sipnet;
+    sipnet.makeCall(intNumber, customerId);
+    time++;
+}
+
+void CanPlant::OfficeWorker::pickCall(int number, float time)
+{
+    //int recordedNumber = number;
+    number = 0;
+    time++;
+}
+
+int CanPlant::OfficeWorker::reserveGood(int customerId, int amount)
+{
+    customerId++;
+    return amount++;
+}
+
+int CanPlant::produceCans(int fish, int tin)
+{
+    int amount = fish / tin;
+    return amount;
+}
+
+void CanPlant::loadBoxes(int boxes, int wareHouseNumber)
+{
+    boxes++;
+    wareHouseNumber++;
+}
+
+double CanPlant::reportScrapOut(double raw, double work)
+{
+    return raw * work *.01;
+}
+
+bool SipProvider::makeCall(int src, int dst)
+{
+    if (src == dst)
+    {
+        return false;
+    }
+    else
+    {
+        return true;
+    }
+}
+
+double SipProvider::chargeCustomer(double time, int customerId)
+{
+    if (customerId == 0)
+    {
+        return 0;
+    }
+    else
+    {
+        return time * price;
+    }
+}
+
+double SipProvider::traficCount(double time)
+{
+    return time * 0.1;
+}
+
+void Cat::eat(char foodType)
+{
+    if (foodType == 'F')
+    {
+        Cat::mew(3);
+    }
```

Comment:
if you're calling the member function `mew`, there is no need to fully qualify the name like this. 
- [ ] revise